sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.1.1 (2023-10-16)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## Features Added
+- Added documentation for using sovereign cloud
```

Comment:
We don't typically ship a new package version for documentation updates. Is there a reason we are doing so here?